test-it/karma/fixtures/http/PutOpenAddMethodClose.via

```diff
@@ -0,0 +1,42 @@
+define(MyVI dv(VirtualInstrument (
+    Locals: c(
+        // Shared
+        e(dv(.UInt32 0) handle)
+        e(c(
+            e(.Boolean status)
+            e(.Int32 code)
+            e(.String source)
+        ) error)
+
+        // OpenHandle parameters
+        e('' cookieFile)
+        e('' username)
+        e('' password)
+        e(dv(.UInt32 1) verifyServer)
+
+        // AddHeader parameters
+        e('' header)
+        e('' value)
+
+        // PUT parameters
+        e('' url)
+        e('' outputFile)
+        e('' buffer)
+        e(10000 timeout)
+        e('' headers)
+        e('' body)
+        e(dv(.UInt32 0) statusCode)
+    )
+    clump (
+        HttpClientOpen(cookieFile username password verifyServer handle error)
+
+        // Added pragma: no-cache so IE11 will not cache the network request
+        HttpClientAddHeader(handle 'pragma' 'no-cache' error)
+
+        HttpClientAddHeader(handle header value error)
+        HttpClientPut(handle url outputFile buffer timeout headers body statusCode error)
+        HttpClientClose(handle error)
+    )
+) ) )
+
+enqueue(MyVI)
```

Comment:
Add new lines at end of file. Probably this is not the only file with this issue.